lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          FACTORY_CALLS = Set[:create, :build, :create_list,
+                              :build_list, :create_pair,
+                              :build_stubbed_list, :atributes_for,
+                              :build_stubbed, :build_pair ].freeze
+
+          def_node_matcher :factory_call, <<-PATTERN
+              (send
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} %FACTORY_CALLS
+              $...)
+          PATTERN
+
+          def on_send(node)
+            return if nested_call?(node)
+
+            factory_call(node) do
+              if node.parenthesized?
+                process_with_parentheses(node)
+              else
+                process_without_parentheses(node)
+              end
+            end
+          end
+
+          def process_with_parentheses(node)
```

Comment:
I wonder if Rubocop's modules `RequireParentheses` / `OmitParentheses` can't be largely reused. At least they handle much more cases - like multiline method calls, for which we'll also need specs

Reply:
Tests have been added